contracts/test/ERC721Test.sol

```diff
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity >=0.8.0;
 
-import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
 
-contract ERC721Test is ERC721 {
+contract ERC721Test is ERC721Enumerable {
```

Comment:
What does this change?

Reply:
exposes totalSupply which is typical
should probably try catch that

Reply:
actually our synthetic already assumes Enumerable so gonna add an assumption to README and ship